E3SM-2-0/piClim-histall_r1i1p1f1.json

```diff
@@ -0,0 +1,105 @@
+{
+  "#note_source_type": "explanation of what source_type is goes here",
+
+  "source_type": "AOGCM AER",
```

Comment:
Based on https://wcrp-cmip.github.io/CMIP6_CVs/docs/CMIP6_experiment_id.html
I think this should be `  "source_type": "AGCM AER",`